translations/scantailor_de.ts

```diff
@@ -291,22 +291,22 @@ zur Auswahl mehrerer Seiten.</translation>
         <location filename="../FixDpiDialog.cpp" line="242"/>
         <location filename="../FixDpiDialog.cpp" line="707"/>
         <source>All Pages</source>
-        <translation type="unfinished">Alles Seiten</translation>
+        <translation type="unfinished">Alle Seiten</translation>
```

Comment:
What about removing `type="unfinished"` for fixed and new translations?

Reply:
Done.